tests/integration/security/rbac/util/rbac_util.go

```diff
@@ -26,17 +26,16 @@ import (
 )
 
 type TestCase struct {
-	Request       connection.Connection
-	ExpectAllowed bool
-	Jwt           string
-	RejectionCode string
+	Request          connection.Connection
+	ExpectedRespCode string
+	Jwt              string
```

Comment:
I think this `Jwt` should be part of the request/connection struct, since it's also needed for authentication tests. Wdygt? @lei-tang @yangminzhu

Reply:
It sounds good to me to place Jwt as part of the Connection struct.

Reply:
If @yangminzhu agrees, we can refactor it in a follow-up PR.

Reply:
yeah, you can refactor it to request